docs/chapter_searching/binary_search.md

```diff
@@ -337,7 +369,8 @@ $$
 === "JavaScript"
 
     ```js title=""
-
+    // JavaScript 中的数字足够大（最大数值为 1.7976931348623157e+308 ）
```

Comment:
My bad, 1.7976931348623157e+308 is not infinite, so we still need to handle the big-number problem. I'll update the code.

Reply:
Hahaha. Actually I think both of the explanations are reasonable.